assets/questions/palindrome.js

```diff
@@ -0,0 +1,159 @@
+  starterCode: {
+    python:
+`def isPalindrome(s):
+    return False
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def createPalindromeString(cls, chr_set, input_size):
+        import random
+        ret = ""
+        while input_size > 0:
+            chr = random.choice(chr_set)
+            if input_size == 1:
+                ret += chr
+                input_size -= 1
+            else:
+                ret = chr + ret + chr
+                input_size -= 2
+        return ret
+
+    @classmethod
+    def countNumberOfCharacters(cls, s):
+        ascii_codes = {i:0 for i in range(255)}
+        for chr in s:
+            ascii_codes[ord(chr)] += 1
+
+        odd_cnt = 0
+        for code in ascii_codes:
+            if ascii_codes[code] % 2 is not 0:
+                odd_cnt += 1
+
+        if len(s) % 2 == 0 and odd_cnt == 0:
+            return True
+        elif len(s) % 2 == 1 and odd_cnt == 1:
+            return True
+        return False
+
+    @classmethod
+    def forgetToIgnoreSpace(cls, s):
+        for i in range(len(s)/2):
+            if s[i] is not s[-i-1]:
+                return False
+        return True
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'For this question, you will implement the isPalindrom function. ',
+        'It takes a sentense as input and return True if the input is a ',
+        'palindrome and False if it is not.'
+      ].join(''),
+      [
+        'A palindrome is a word, phrase, or sequence that reads the same ',
+        'backward as forward, e.g., \'madam\' or \'nurses run\'.'
+      ].join(''),
+      'Note that the input string will consist of lowercase letters (a-z) and spaces.'
+    ],
+    prerequisiteSkills: ['String Manipulation'],
+    acquiredSkills: ['String Manipulation'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'isPalindrome',
+    correctnessTests: [{
+      input: '',
+      allowedOutputs: [true]
+    }, {
+      input: 'nurses run',
+      allowedOutputs: [true]
+    }, {
+      input: 'a b   cba',
+      allowedOutputs: [true]
+    }, {
+      input: 'abcdcba',
+      allowedOutputs: [true]
+    }, {
+      input: 'abccba',
+      allowedOutputs: [true]
+    }, {
+      input: 'abab',
+      allowedOutputs: [false]
+    }, {
+      input: 'stressed desserts',
+      allowedOutputs: [true]
+    }, {
+      input: 'dioramas is samaroid',
+      allowedOutputs: [false]
+    }, {
+      input: 'dioramas is si samaroid',
+      allowedOutputs: [true]
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.countNumberOfCharacters',
+      messages: [
+        "Try running your code on 'abab' on paper. Did you expet that result?",
+        [
+          'Are you sure the input string is palindrome? \'abab\', for instance, is not.'
+        ].join(''),
+        [
+          "number of characters in the input string and returning true if ",
```

Comment:
I think you're missing part of the sentence here. Make sure you're double-checking before you push.